third_party/move/move-bytecode-verifier/src/features.rs

```diff
@@ -102,13 +102,18 @@ impl<'a> FeatureVerifier<'a> {
     }
 
     fn verify_function_handles(&self) -> PartialVMResult<()> {
-        if !self.config.enable_resource_access_control {
+        if !self.config.enable_resource_access_control || !self.config.enable_function_values {
```

Comment:
Why these are bundled together?

Reply:
What if one feature is off and the other is on? This will be incorrect then. If `enable_function_values` is true, but `enable_resource_access_control` is false, you will reject code if there is any function attribute

Reply:
Ah, good catch. Fixed.